src/main/java/com/metamx/collections/bitmap/RoaringBitmapFactory.java

```diff
@@ -85,14 +85,8 @@ public ImmutableBitmap makeImmutableBitmap(MutableBitmap mutableBitmap)
       throw new IllegalStateException(String.format("Cannot convert [%s]", mutableBitmap.getClass()));
     }
     try {
-      final ByteArrayOutputStream out = new ByteArrayOutputStream();
-      ((WrappedRoaringBitmap) mutableBitmap).getBitmap().serialize(new DataOutputStream(out));
-      final byte[] bytes = out.toByteArray();
-
-
-      ByteBuffer buf = ByteBuffer.wrap(bytes);
       return new WrappedImmutableRoaringBitmap(
-          new ImmutableRoaringBitmap(buf)
+          new ImmutableRoaringBitmap(ByteBuffer.wrap(mutableBitmap.toBytes()))
```

Comment:
we should make sure toBytes actually creates a copy, otherwise our bitmap won't really be immutable.